src/pages/AddVacation/AddVacation.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { Dayjs } from "dayjs";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import { VacationDatePicker } from "../../shared/DatePicker";
+import { VacationSelector } from "../../shared/VacationSelector";
+import { styledAddVacation } from "./style";
+
+const AddVacation = () => {
+  const [vacationType, setVacationType] = useState<string | null>("");
+  const [startDate, setStartDate] = useState<Dayjs | null>(null);
+  const [endDate, setEndDate] = useState<Dayjs | null>(null);
+  const [notes, setNotes] = useState<string | null>();
+  const [error, setError] = useState<string>("");
+
+  const isFildesEmpty = startDate && endDate && vacationType;
+
+  const addNewVacation = () => {
+    if (!isFildesEmpty) return console.log("Error");
+    console.log({ vacationType, endDate, startDate });
+  };
+
+  return (
+    <>
+      <Box sx={styledAddVacation.vacationWrapper}>
+        <Box>
+          <Box sx={styledAddVacation.datePickBlock}>
+            <VacationSelector
```

Comment:
Put the VacationSelector component as a subfolder in the AddVacation folder. This component is specific only for AddVacation.